### PR TITLE
[e2e] Remove DNS test for invalid URL

### PR DESCRIPTION
**Pull request by jrvaldes**
This commit removes the DNS invalid URL test because it succeeds even if the request does not hit the DNS server.

Follow-up to https://github.com/openshift/windows-machine-config-operator/commit/c6092e2be3279651c8d4c57a9d48817f7cf07b78

### CODE REVIEW

**Review by selansen — CHANGES_REQUESTED** [review]
Now that we removed invalid test case, Can you remove the valid URL from the other test case?
We can keep it testPodDNSresolution

**Comment by jrvaldes** [comment]
> Now that we removed invalid test case, Can you remove the valid URL from the other test case? We can keep it testPodDNSresolution

That's a good one, will adjust.

I considered that before pushing the commit. The reason I did not, was to keep a two-level structure for `testPodDNSResolution` tests, where `Valid URL` is the actual test. In case, we need/want to add more PodDNSResolution' related tests in the future.

**Comment by jrvaldes** [comment]
test /lint

File: test/e2e/network_test.go
Comment (sebsoto) [inline test/e2e/network_test.go:188]: `...because it succeeds even if the request does not hit the DNS server.`
Two things:
1) Why does it always succeed? I would think curl would return an error if the site doesnt exist.
2) Is that the only justification for removing this test? Is there any value in fixing this test?

**Review by selansen — APPROVED** [review]
/lgtm

**Comment by jrvaldes** [comment]
/test aws-e2e-upgrade
/test azure-e2e-operator

Cluster installation failed.

**Comment by aravindhp** [comment]
/hold

until #868 merges

**Comment by wking** [comment]
openshift/cluster-version-operator#728 has landed, hopefully addressing [this reported overrides issue][1].

/retest

[1]: https://bugzilla.redhat.com/show_bug.cgi?id=2042039

**Comment by aravindhp** [comment]
@wking not sure if the retests will actually help given #869 has merged. We might need to revert that to actually test openshift/cluster-version-operator#728. cc: @saifshaikh48

**Comment by jrvaldes** [comment]
#868 is closed.

**Comment by jrvaldes** [comment]
/hold

until https://github.com/openshift/windows-machine-config-operator/pull/873 merges

**Comment by jrvaldes** [comment]
/hold cancel

#873  merged

**Comment by jrvaldes** [comment]
> @jrvaldes: #863 failed to apply on top of branch "community-4.8":
> 
> ```
> Applying: Remove DNS test for invalid URL
> Using index info to reconstruct a base tree...
> M	test/e2e/network_test.go
> Falling back to patching base and 3-way merge...
> Auto-merging test/e2e/network_test.go
> CONFLICT (content): Merge conflict in test/e2e/network_test.go
> error: Failed to merge in the changes.
> hint: Use 'git am --show-current-patch=diff' to see the failed patch
> Patch failed at 0001 Remove DNS test for invalid URL
> When you have resolved this problem, run "git am --continue".
> If you prefer to skip this patch, run "git am --skip" instead.
> To restore the original branch and stop patching, run "git am --abort".
> ```

Needs https://github.com/openshift/windows-machine-config-operator/pull/877 to merge.

**Comment by jrvaldes** [comment]
`cherry-pick release-4.9`  needs https://github.com/openshift/windows-machine-config-operator/pull/855 to merge

**Comment by jrvaldes** [comment]
`cherry-pick release-4.8` needs https://github.com/openshift/windows-machine-config-operator/pull/878 to merge

**Comment by jrvaldes** [comment]
> > @jrvaldes: #863 failed to apply on top of branch "community-4.8":
> > ```
> > Applying: Remove DNS test for invalid URL
> > Using index info to reconstruct a base tree...
> > M	test/e2e/network_test.go
> > Falling back to patching base and 3-way merge...
> > Auto-merging test/e2e/network_test.go
> > CONFLICT (content): Merge conflict in test/e2e/network_test.go
> > error: Failed to merge in the changes.
> > hint: Use 'git am --show-current-patch=diff' to see the failed patch
> > Patch failed at 0001 Remove DNS test for invalid URL
> > When you have resolved this problem, run "git am --continue".
> > If you prefer to skip this patch, run "git am --skip" instead.
> > To restore the original branch and stop patching, run "git am --abort".
> > ```
> 
> Needs #877 to merge.

/cherry-pick community-4.8

#877 merged

**Comment by jrvaldes** [comment]
/cherry-pick release-4.9

https://github.com/openshift/windows-machine-config-operator/pull/855 has merged

**Comment by jrvaldes** [comment]
/cherry-pick release-4.8

https://github.com/openshift/windows-machine-config-operator/pull/878 has merge

**Comment by jrvaldes** [comment]
/cherry-pick release-4.8 

https://github.com/openshift/windows-machine-config-operator/pull/878 has merge

**Pull request merged**
